kong/pdk/request.lua

```diff
@@ -28,6 +28,33 @@ local PHASES = phase_checker.phases
 cjson.decode_array_with_array_mt(true)
 
 
+do -- add patch for faster ngx.req.get_header
+  local req_get_headers = ngx.req.get_headers
+  -- use ngx.ctx as weak cache key
+  local req_get_headers_cache = setmetatable({}, {__mode="k"})
```

Comment:
needs some space, like `__mode = "k"`,
also, there should be a `,` at the end of each item.